pkg/providers/v1/aws.go

```diff
@@ -239,6 +239,9 @@ const volumeAttachmentStuck = "VolumeAttachmentStuck"
 // Indicates that a node has volumes stuck in attaching state and hence it is not fit for scheduling more pods
 const nodeWithImpairedVolumes = "NodeWithImpairedVolumes"
 
+// Allows the user to control the load balancer name from an annotation
+const ServiceAnnotationLoadBalancerName = "service.beta.kubernetes.io/aws-load-balancer-name"
```

Comment:
can you include more context in PR description about this change. Like are we trying to respect the same annotation as https://github.com/kubernetes-sigs/aws-load-balancer-controller? Or just coincidence it's the same?

Reply:
Yeah its intentional that we are using the same annotation.